app/controllers/concerns/report_actions.rb

```diff
@@ -70,8 +70,9 @@ def generate_experiment_contents_json(experiment, selected_modules)
   def generate_module_contents_json(my_module)
     res = []
     ReportExtends::MODULE_CONTENTS.each do |contents|
-      protocol = contents.element == :step ? my_module.protocol.present? : true
-      next unless in_params?("module_#{contents.element}".to_sym) && protocol
+      protocol = contents.element == :steps ? my_module.protocol.present? : true
```

Comment:
this was corrected because it didn't work, but now it didn't work because it was changed :D. Anyway, this should be correct.